Makefile

```diff
@@ -1,14 +1,9 @@
-contracts:
-	abigen --abi=contracts/rollup/rollup.abi --pkg=rollup --out=contracts/rollup/rollup.go
-	abigen --abi=contracts/registry/registry.abi --pkg=registry --out=contracts/registry/registry.go
-	abigen --abi=contracts/state/state.abi --pkg=state --out=contracts/state/state.go
-	abigen --abi=contracts/depositmanager/depositmanager.abi --pkg=depositmanager --out=contracts/depositmanager/depositmanager.go
-	abigen --abi=contracts/tokenregistry/tokenregistry.abi --pkg=tokenregistry --out=contracts/tokenregistry/tokenregistry.go
-	abigen --abi=contracts/burnauction/burnauction.abi --pkg=burnauction --out=contracts/burnauction/burnauction.go
-	abigen --abi=contracts/transfer/transfer.abi --pkg=transfer --out=contracts/transfer/transfer.go
-	abigen --abi=contracts/massmigration/massmigration.abi --pkg=massmigration --out=contracts/massmigration/massmigration.go
-	abigen --abi=contracts/create2transfer/create2transfer.abi --pkg=create2transfer --out=contracts/create2transfer/create2transfer.go
-	abigen --abi=contracts/accountregistry/accountregistry.abi --pkg=accountregistry --out=contracts/accountregistry/accountregistry.go
+build-hubble-contracts:
```

Comment:
This introduces a `nodejs` requirement for `commander` , right? Is there any way to avoid it?

Reply:
only when we need to generate the abi binding, which happens when we bump contract submodule version.
Building commander doesn't require nodejs.

Reply:
Nice!